vars/RedHat.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
If you don't use it. You don't have to have it.

Reply:
เหมือนจะพังถ้าไม่ใส่อ่ะครับ มันให้เพิ่ม flag skip ถ้าไม่มีของ family นั้นๆ เอาแบบนั้นไหมครับ